snooty/parser.py

```diff
@@ -159,6 +159,10 @@ def __call__(
         try:
             checksum = self.asset.get_checksum()
             options["checksum"] = checksum
+            dimensions = self.asset.dimensions
+            if dimensions is not None:
+                options["width"] = str(dimensions[0])
```

Comment:
@i80and seems like this will overwrite some input user data (ie. width option). validating this is None before overwriting in a follow up PR